libs/librepcb/common/bom/bomcsvwriter.cpp

```diff
@@ -50,13 +50,18 @@ QList<QStringList> BomCsvWriter::toStringList(const Bom& bom) noexcept {
   QList<QStringList> lines;
   // Don't translate the CSV header to make BOMs independent of the user's
   // language.
-  lines.append(QStringList{"Quantity", "Designators"} + bom.getColumns());
+  QStringList header = QStringList{"Quantity", "Designators"} + bom.getColumns();
+  QStringList headerCols;
+  foreach (const QString& headerItem, header) {
+    headerCols += '"' + headerItem + '"';
+  }
+  lines.append(headerCols);
   foreach (const BomItem& item, bom.getItems()) {
     QStringList cols;
     cols += QString::number(item.getDesignators().count());
-    cols += cleanStr(item.getDesignators().join(", "));
+    cols += '"' + cleanStr(item.getDesignators().join(", ")) + '"';
```

Comment:
Will `cleanStr` ensure that all quotes inside the string are escaped? Otherwise we'll end up with broken CSV files if any of the strings contain quotes.

It might also be nice if double quotes are only added when necessary (i.e. when the value contains a comma).